src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -23,6 +23,7 @@ trait Create
      */
     public function create($data)
     {
+        //dd($data);
```

Comment:
There's a dd() here.

Reply:
Don't get mad at me, I deleted around 50 `dd` blocks :)

This branch needs a clean up, but what I really wanted was for you to give it a look in the implemented solutions for the problems.

You already found 2 bugs that i missed, they are fixed.

I will write the doc block/ some inline comments when we get into the final solution :)

Reply:
Ahaha. I understand. Best to include _some_ comments for methods, though - otherwise it takes a lot of time and effort to understand the code, and put together a proper review.